test/models/Applicant/Link/Model.test.ts

```
@@ -50,6 +49,15 @@ describe("ApplicantLink model", () => {
     await expect(applicantLink.save()).rejects.toThrow();
   });
 
+  it("raise an error if invalid url is provided", async () => {
```

lo escribiria  como: `"should raise an error if invalid url is provided"`